shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -671,21 +671,27 @@ - (void)viewDidLoad {
 }
 
 - (void)addInternalPlugins {
-  [self.keyboardManager release];
-  self.keyboardManager = [[FlutterKeyboardManager alloc] init];
+  FlutterKeyboardManager* keyboardManager = [[FlutterKeyboardManager alloc] init];
+  self.keyboardManager = keyboardManager;
+  [keyboardManager release];
+  fml::WeakPtr<FlutterViewController> weakSelf = [self getWeakPtr];
   FlutterSendKeyEvent sendEvent =
       ^(const FlutterKeyEvent& event, FlutterKeyEventCallback callback, void* userData) {
-        [_engine.get() sendKeyEvent:event callback:callback userData:userData];
+        [weakSelf.get()->_engine.get() sendKeyEvent:event callback:callback userData:userData];
```

Comment:
This didn't show up in the linter, but this retain cycle was exposed when I wrote the `testReleasesKeyboardManagerOnDealloc` and the view controller wasn't being released.